examples/nodejs/lambda-examples/cloudwatch-metrics/lambda/handler.ts

```diff
@@ -0,0 +1,86 @@
+import {GetSecretValueCommand, SecretsManagerClient} from '@aws-sdk/client-secrets-manager';
+import {
+  CacheClient,
+  Configurations,
+  DefaultMomentoLoggerFactory,
+  DefaultMomentoLoggerLevel,
+  CredentialProvider,
+} from '@gomomento/sdk';
+import {ExperimentalMetricsLoggingMiddleware} from '@gomomento/sdk/dist/src/config/middleware/experimental-metrics-logging-middleware';
+
+const _secretsClient = new SecretsManagerClient({});
+const _cachedSecrets = new Map<string, string>();
+let _cacheClient: CacheClient | undefined = undefined;
+
+export const handler = async () => {
+  try {
+    const loggerFactory = new DefaultMomentoLoggerFactory(DefaultMomentoLoggerLevel.DEBUG);
```

Comment:
do we log a ton when debug logging? If we do we might want to make this `INFO`. If not then `DEBUG` is fine

Reply:
O is the `DEBUG` for the middleware, so we can grep the logs and make a dashboard